test/configCases/plugins/banner-plugin-hashing/index.js

```diff
@@ -0,0 +1,35 @@
+const parseBanner = (banner) => {
+	return banner
+		.slice(4,-3)
+		.split(", ")
+		.map(n => n.split(":"))
+		.reduce((acc, val) => {
+			acc[val[0]] = val[1];
+			return acc;
+		}, {});
+};
+
+var source = require("fs")
+	.readFileSync(__filename, "utf-8")
+	.split("\n")
+	.slice(0,1)[0];
+
+const banner = parseBanner(source)
+
+it("should interpolate file hash in bundle0 chunk", () => {
+	banner["hash"].should.not.equal("[hash]");
```

Comment:
please make the tests more strict. This test would also pass if all placeholders are replaced with `""`.